bears/go/GoErrCheckBear.py

```diff
@@ -0,0 +1,18 @@
+from coalib.bearlib.abstractions.Linter import linter
+
+
+@linter(executable='errcheck',
+        output_format='regex',
+        output_regex=r'(?P<filename>[^:]+):(?P<line>\d+):'
+        r'(?P<col>\d+)\s*(?P<message>.*)',
+        result_message="Erroneous function calls detected by errcheck")
+class GoErrCheckBear:
+    """
+    Checks the code for all function calls that have unchecked errors.
+    GoErrCheckBear runs ``errcheck`` over each file to find such functions.
+    """
+    LANGUAGES = "Go"
+
+    @staticmethod
+    def create_arguments(filename, file, config_file):
```

Comment:
`errcheck` takes in a few arguments. Implementing them as settings will make this bear more complete.